p2d/bin/p2d

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+export PYTHONPATH="$(dirname "$(dirname "$(readlink -f "$0")")"):$PYTHONPATH"
+export CONFIG_PATH="$(dirname "$(dirname "$(readlink -f "$0")")")"
+export TESTLIB_PATH=$CONFIG_PATH
+export LOG_PATH='~/.p2d'
+exec python -m p2d "$@"
```

Comment:
Seems like `python3` should be used. My Ubuntu 20.04 WSL doesn't contains `python`, and my Ubuntu 18.04 VM instances defaults to py2.

Reply:
@xzm2000 Do you have any suggestions? Something like `/usr/bin/env python3`?